.github/workflows/diff.yml

```diff
@@ -31,8 +30,12 @@ jobs:
         run: docker pull obolibrary/odklite
         if: steps.check.outputs.triggered == 'true'
       # Checks-out current branch
+      - uses: xt0rted/pull-request-comment-branch@v1
```

Comment:
`xt0rted` -> be careful with re-using actions that you do not 100% trust.. Make sure all actions you use are safe!

Reply:
For example, this one has too few stars to be trustworthy. They could include malware in their action, or read our tokens or what not - at least ensure this cannot be the case.

Reply:
I dont know either, sorry.

Reply:
I can do what deepak suggested and clone it into my repo and use that, but I don't think that's safe too, it could still redirect in the actions to redirect to a remote thing. The ideal situation would be to manually do the actions by scanning the code and copying it over I guess?

Reply:
TypeScript is not my forte, but for what it’s worth, I see nothing nefarious in `xt0rted/pull-request-comment-branch@v1`… The code is fairly small (too small to hide anything), and it only depends on modules provided by GitHub itself.